examples/dt/dcn/values.yaml.j2

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+data:
+    customDataplanService:
+      name: nova-custom-ceph-{{ _az }}
+    nodeset_name: {{ _group_name }}-edpm
+    ceph_conf:
+{% for _file in _ceph_files %}
+        {{ _file | basename }}: {{ lookup('file', _file, rstrip=False) | b64encode }}
+{% endfor %}
+    nodeset:
+        services:
+        - bootstrap
+        - configure-network
+        - validate-network
+        - install-os
+        - ceph-hci-pre
+        - configure-os
+        - ssh-known-hosts
+        - run-os
+        - reboot-os
+        - install-certs
+        - ceph-client
+        - ovn
+        - neutron-metadata
+        - libvirt
+        - nova-custom-ceph-{{ _az }}
+    nova:
+        ceph:
+            conf: |
+                [libvirt]
+                images_type=rbd
+                images_rbd_pool=vms
+                images_rbd_ceph_conf=/etc/ceph/{{ cifmw_ceph_client_cluster }}.conf
+                images_rbd_glance_store_name=default_backend
```

Comment:
I believe It should be:
`                images_rbd_glance_store_name={{ cifmw_ceph_client_cluster }}`

For example for az1:
 images_rbd_glance_store_name=az1

Reply:
Done